src/index.js

```diff
@@ -106,11 +106,11 @@ function ParseServer({
     passwordResetSuccess: undefined
   },
 }) {
-  
+  setFeature('serverVersion', parseServerPackage.version);
```

Comment:
Then we can include things like installed packages, adapters and their version, and so on